### ISSUE TITLE
call the method "Py_DecodeLocale" get an error !

### ISSUE THREAD

**Issue by wanghe826** [dimaqq/pios#1]
Hi, thank you for your open source code !  but i got an error when excute your Example project. 
below is the error:

<img width="1544" alt="2017-02-24 5 39 43" src="https://cloud.githubusercontent.com/assets/5173739/23298498/df8fca7a-fab8-11e6-815e-872f52c6e845.png">

and what's the "xz.framework"??

**Comment by dimaqq** [comment]
Hi, I think that you have wrong version of Python framework unpacked.
https://docs.python.org/3/c-api/sys.html#c.Py_DecodeLocale is available from Python 3.5

Perhaps I forgot to mention this in the setup steps.

Grab prebuilt frameworks from:
https://github.com/pybee/Python-Apple-support/tree/3.5#quickstart

Note:
* branch corresponds to Python version (i.e. use 3.5)
* package corresponds to target device (iOS for iPhone)

You'll get a file like `Python-3.5-iOS-support.b2.tar.gz` (patch version may vary)

Then unpack it in the top level and you'll get something like this:
```
⋊> ~/i/playing-with-python-on-ios on master ◦ ls -la
total 24
drwxr-xr-x  14 dima  staff  476 18 Feb 08:11 .
drwxr-xr-x   4 dima  staff  136 17 Feb 19:06 ..
drwxr-xr-x  16 dima  staff  544 24 Feb 13:19 .git
-rw-r--r--   1 dima  staff  485  8 Feb 21:14 .gitignore
drwxr-xr-x@  5 dima  staff  170  5 Sep 10:33 BZip2.framework
drwxr-xr-x@  5 dima  staff  170 16 Feb 18:21 Dishuhu Malala Arkuyaya.xcodeproj
drwxr-xr-x@  5 dima  staff  170  5 Sep 10:46 OpenSSL.framework
drwxr-xr-x@  6 dima  staff  204  5 Sep 11:01 Python.framework
-rw-r--r--   1 dima  staff  839 18 Feb 08:11 README.md
drwxr-xr-x@  5 dima  staff  170  5 Sep 10:37 XZ.framework
drwxr-xr-x   4 dima  staff  136  7 Feb 22:06 app
drwxr-xr-x   8 dima  staff  272 18 Feb 12:12 app_packages
drwxr-xr-x   7 dima  staff  238  8 Feb 21:05 dimark
-rw-r--r--@  1 dima  staff   32  5 Sep 11:01 support.version
```

Frameworks are:
* bzip2 and xz — compression algorithm
* openssl — Python is linked to it, so :)
* Python — you need it

**Comment by dimaqq** [comment]
@wanghe826 I wonder if you got anywhere yet...

I've just discovered that Python dependencies were missing from the repo and I've added them now.
